core/Access.php

```diff
@@ -336,19 +336,29 @@ public function checkUserHasSuperUserAccess()
     }
 
     /**
-     * If the user doesn't have an ADMIN access for at least one website, throws an exception
+     * Returns `true` if the current user has admin access to at least one site.
      *
-     * @throws \Piwik\NoAccessException
+     * @return bool
      */
-    public function checkUserHasSomeAdminAccess()
+    public function isUserHasSomeAdminAccess()
```

Comment:
I added it to this class for easier injecting of dependencies and easier testing. Added tests for this method... this is what we should use in the future anyway instead of `Piwik::check...` or `Piwik::is...`